src/database/schema/UserSchema.ts

```diff
@@ -191,6 +199,7 @@ class UserSchema implements BaseDbSchema {
       const query = `SELECT * FROM ${UserSchema.getTableName()} WHERE user_id = ? AND channel_id = ? LIMIT 1`;
       tx.executeSql(query, [this.userId, this.channelId], (isExistsTx, results) => {
         const isExists = results?.rows?.length > 0;
+        console.log('isExists', this.userId, isExists);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Logging sensitive data like userId can lead to potential security risks. Consider removing or obfuscating such data if it's not necessary for debugging purposes.
```diff
-        console.log('isExists', this.userId, isExists);
+        console.log('isExists', '***', isExists); // Obfuscate sensitive data
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->